### PR TITLE
Sash/footer

### PR DESCRIPTION
**Pull request by binary-sash**

### CODE REVIEW

File: src/components/layout/footer.js [hunk 1]
```diff
@@ -1,9 +1,233 @@
...
+                                <StyledLink
+                                    activeClassName="active"
+                                    to="/"
+                                    aria-label={localize('Responsible Trading')}
+                                >
+                                    {localize('Responsible Trading')}
+                                </StyledLink>
+                            </div>
+                            <Social>
+                                <p>{localize('folow us on')}</p>
```
Comment (ashkanx) [inline src/components/layout/footer.js]:
```suggestion
                                <p>{localize('follow us on')}</p>
```

Comment (ahkuma) [reply src/components/layout/footer.js]: I think using a h4 would look nice.

File: src/components/layout/footer.js [hunk 2]
```diff
@@ -1,9 +1,233 @@
...
+import Logo from '../../images/svg/logo.svg'
+import YouTube from "../../images/svg/youtube.svg";
+import Twitter from "../../images/svg/twitter.svg";
+import Telegram from "../../images/svg/telegram.svg";
+import Reddit from "../../images/svg/reddit.svg";
+import Facebook from "../../images/svg/social-facebook.svg";
+import Labuan from "../../images/svg/footer-labuan.svg";
+import FSC from "../../images/svg/fsc.svg";
+import Vanuatu from "../../images/svg/footer-vanuatu.svg";
+import Warning from "../../images/svg/warning.svg";
```
Comment (OskarAhl) [inline src/components/layout/footer.js]: Absolute imports:

```suggestion
import Warning from "images/svg/warning.svg";
```

File: src/components/layout/footer.js [hunk 3]
```diff
@@ -1,9 +1,233 @@
...
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    width: 100%;
+
+    @media ${device.laptop} {
+        grid-template-columns: 1fr;
+        grid-row-gap: 2rem;
+    }
+`
+const FooterNav = styled.div`
```
Comment (OskarAhl) [inline src/components/layout/footer.js]:
```suggestion
const FooterNav = styled.nav`
```

File: src/components/layout/footer.js [hunk 4]
```diff
@@ -1,9 +1,233 @@
...
+    }
+    h4 {
+        text-transform: uppercase;
+        font-size: 1.4rem;
+        font-weight: bold;
+        letter-spacing: 1px;
+        margin-bottom: 1.1rem;
+    }
+`
+const StyledLink = styled(props => <Link {...props} />)`
```
Comment (OskarAhl) [inline src/components/layout/footer.js]: Please use LocalizedLink instead (will be necessary once we enable routing for languages)
```suggestion
const StyledLink = styled(props => <LocalizedLink {...props} />)`
```

File: src/images/svg/mastercard.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="84" height="64" viewBox="0 0 84 64">
```
Comment (OskarAhl) [inline src/images/svg/mastercard.svg]: please add `aria-labelledby="..."` + `<title>descriptive title<title/>` for all the svgs (see the other svgs for reference)

File: src/images/svg/footer-labuan.svg [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="113" height="45" viewBox="0 0 113 45">
```
Comment (OskarAhl) [inline src/images/svg/footer-labuan.svg]: can you add `title` and `aria-labelledby` to this one too

File: src/images/svg/logo.svg [hunk 7]
```diff
@@ -1,4 +1,5 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="38" height="36" viewBox="0 0 38 36">
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="36" viewBox="0 0 38 36" aria-labelledby="footerlogo-icon-title">
+    <title id='footerlogo-icon-title'>Deriv icon</title>
```
Comment (ahkuma) [inline src/images/svg/logo.svg]: The other icons is consistent in file name, title text and ID, this one is a bit weird. Could we possible have something like `deriv.svg`, `deriv-icon-title` and `Deriv icon` or `deriv-footer.svg`, `deriv-footer-icon-title` and `Deriv icon`?

**Pull request merged**